velox/functions/prestosql/Comparisons.h

```diff
@@ -26,15 +26,14 @@ template <typename T>
 struct TimestampWithTimezoneComparisonSupport {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
-  // Convert TimestampWithTimezone from original timezone to GMT in milliseconds
+  /// Timestamp with time zone values contain the milliseconds
+  /// already adjusted to UTC/GMT.
+  /// That means they are already normalized on UTC. We can directly compare
+  /// the UTC seconds and don't need to convert anything.
```

Comment:
Seems like TimestampToTimezone Java code is also simply unpacking the Ms value for each operation https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/type/TimestampWithTimeZoneOperators.java#L69 and there isn't a timezone conversion.

I see timezone conversion for other non-timezone based time values dependent on config [adjust_session_timestamp_to_timezone]( https://github.com/facebookincubator/velox/blob/main/velox/core/QueryConfig.h#L76). But that should not be applicable here.